### ISSUE TITLE
Mediation Problem

### ISSUE THREAD

**Issue by mememto** [mobfox/MobFox-iOS-SDK#47]
Hi, I'm using Admob to show MobFox ads (banner and interstitial). Testing on iOS 11 devices with Xcode 9 GM the app crashes with the following message:

```
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Completion handler passed to -[MobFoxTagAd webView:decidePolicyForNavigationAction:decisionHandler:] was called more than once'
*** First throw call stack:
(0x184e47d38 0x18435c528 0x184e47c80 0x193e6cbdc 0x193eb1700 0x104f30c6c 0x193eaf254 0x1940171b4 0x19403eac8 0x194037bf4 0x193eacba8 0x1940898b4 0x193e6f8b0 0x193e6d0d4 0x193e6fae0 0x18c074d10 0x18c074fd0 0x184df0358 0x184df02d8 0x184defb60 0x184ded738 0x184d0e2d8 0x186b9ff84 0x18e2ba880 0x1046f39e0 0x18483256c)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

I think that this was already mentioned but it seems that It was not solved :/

**Comment by nabriski** [comment]
Hi,

Actually this is new, the last one was on a slightly different path as we now use WKWebView for the banners.
Do you have a specific scenario to reproduce this?

Thanks

**Comment by nabriski** [comment]
Found it. We'll issue a bug fix version on Sunday.

**Comment by mememto** [comment]
Great, thanks for your efforts =)

**Comment by nabriski** [comment]
fixed in 3.1.8c
